src/schema/schema.ts

```diff
@@ -41,7 +41,6 @@ import { ZimbraSchemaOptions } from './types';
 
 import { ZimbraBatchClient } from '../batch-client';
 import { coerceBooleanToString } from '../utils/coerce-boolean';
-import { ZimbraNotifications } from './notifications';
```

Comment:
should we delete this file as well?

Reply:
Yep that was the last step to do, I will just remove it...

Reply:
Done - deleted